simulators/nav/src/components/NavSimulator.vue

```diff
@@ -454,7 +454,7 @@ export default class NavSimulator extends Vue {
 
     /* Set up publishing LCMs */
     this.intervalLcmPublish = window.setInterval(() => {
-      this.publish('/auton', { type: 'AutonState', is_auton: this.autonOn }, false);
+      this.publish('/auton_enabled', { type: 'Enable', is_auton: this.autonOn }, false);
```

Comment:
Should be `enabled` and not `is_auton` @ankithu @danieljiang520 

@danieljiang520 along with that, you also want to change the `is_auton` found `simulators/nav/src/.eslintrc.js`, it's located twice in there, and note that the order of the variables is alphabetical which we'll want to keep